## Quiz1
This is a Quiz. Please write a suitable program according to the instructions. You may see the tutorials or search on the Internet.

Create a variable called ```original``` and initialize it with whatever value you choose. (e.g. -321012.34)

In [45]:
original=-321012.34

Print **"The value inside original is ○○"**.<br>
When printing, concatenate the string "The value inside original is " and ```original```. Note that in order to concatenate string and float, you must firstly convert ```original``` to string.<br>
(Results will be like→The value inside original is -321012.34)

In [46]:
print("The value inside original is " + str(original))

The value inside original is -321012.34


Next, write if-else statement.  
```python
if(CONDITION):
    #Create the variable called "number" and initialize it with original
else:
    #Create the variable called "number" and initialize it with -original
```
>Repleace CONDITION with the program meaning "Is original a positive value?".  
>Repleace "#Create the variable......" with the code.

In [47]:
if(original > 0):
    number=original
else:
    number=-original

Print the ```number```. It must be bigger than or equal to 0.<br>
(Results will be like→321012.34)

In [48]:
print(number)

321012.34


Next, overwrite ```number``` with \"```number``` rounded down to the nearest decimal\".<br>
This can be done by casting float to integer. (Just like you cast string to integer)

In [49]:
number=int(number)

Print the number. It must be rounded down to the nearest decimal<br>
(Results will be like→321012)

In [50]:
print(number)

321012


Print the type of ```number```. It must be <class 'int'>.

In [51]:
print(type(number))

<class 'int'>


Next, overwrite number with "```number```\'s last three digits", and print it.  
This can be done by calculating the reminder of number÷1000.  <br>
(Results will be like→12)

In [52]:
number%=1000
print(number)

12


Next, write if statement.  
```python
if(CONDITION):
    #Add 100 to number
```
>Repleace CONDITION with the program meaning "is the ```number``` less than 100?".  
>Repleace "#Add 100 to number" with the code.

In [53]:
if(number<100):
    number+=100

Print the number.<br>
(Results will be like→112)

In [54]:
print(number)

112


Create a **string variable** called ```text``` and initialize it with ```number```. Remember to cast int to string.

In [55]:
text=str(number)

Print the type of ```text```. It must be <class 'str'>.

In [56]:
print(type(text))

<class 'str'>


Create a **string variable** called ```text2``` and initialize it with ```text*2```. Print ```text2```.<br>
(Results will be like→112112)

In [57]:
text2=text*2
print(text2)

112112


Create an **integer variable** called ```number2``` and initialize it with ```text2```. Remember to cast string to int.  
After that, print the type of ```number2```. The result must be <class 'int'>

In [58]:
number2=int(text2)
print(type(number2))

<class 'int'>


Divide and assign ```number2``` with 7 and print it.<br>
(Results will be like→16016.0)

In [59]:
number2/=7
print(number2)

16016.0


Divide and assign ```number2``` with 11 and print it.<br>
(Results will be like→1456.0)

In [60]:
number2/=11
print(number2)

1456.0


Divide and assign ```number2``` with 13 and print it.<br>
(Results will be like→112)

In [61]:
number2/=13
print(number2)

112.0


Finally, run the following code.
```python
print(number==number2)
```
If "True" is printed, you are all done!

In [62]:
print(number==number2)

True
